Formula/lua@5.1.rb

```diff
@@ -40,14 +40,17 @@ class LuaAT51 < Formula
     end
   end
 
+  # Don't use the https://luarocks.org/releases/luarocks-x.y.z.tar.gz URL
```

Comment:
comment looks wrong

Reply:
Oh, maybe they've fixed the redirect now. I'll check.

Reply:
Still there:
```
Resource: luarocks
==> Downloading https://luarocks.org/releases/luarocks-2.4.4.tar.gz
==> Downloading from http://luarocks.github.io/luarocks/releases/luarocks-2.4.4.tar.gz
/usr/bin/curl --show-error --user-agent Homebrew/1.5.12-9-g49927ed (Macintosh; Intel Mac OS X 10.13.4) curl/7.54.0 --fail --location --remote-time --continue-at - --output /usr/local/var/homebrewcache/lua--luarocks-2.4.4.tar.gz.incomplete http://luarocks.github.io/luarocks/releases/luarocks-2.4.4.tar.gz
```

Reply:
have you reported it?

Reply:
I can go one better and file a PR, tbh. It's a one-liner.

https://github.com/luarocks/luarocks-site/blob/b8ade0ddf390eb40631f91d57ecf4f35595858d9/nginx.conf#L102

Reply:
https://github.com/luarocks/luarocks-site/pull/131